docs/pages/includes/install-linux.mdx

```diff
@@ -63,6 +69,20 @@
         ```
   </TabItem>
 
+  <TabItem label="Debian/Ubuntu Legacy (DEB)">
+        ```code
```

Comment:
Nit: Add a note on this tab explaining deficiencies of these repos and that we recommend using the new ones.